spec/controllers/home_controller_spec.rb

```diff
@@ -1,2 +1,82 @@
 require 'rails_helper'
 
+RSpec.describe HomeController, type: :controller do
+  describe 'GET #index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.